packages/react/src/components/Comments/ThreadsSidebar.tsx

```diff
@@ -49,47 +55,74 @@ export function ThreadsSidebar(props: {
   filter?: "open" | "resolved";
   // TODO: Should be implemented for both floating and sidebar views
   // maxCommentsPerThread?: number;
-  sort?: "position" | "newest" | "replies";
+  sort?: "position" | "newest" | "oldest";
 }) {
   const editor = useBlockNoteEditor();
 
   if (!editor.comments) {
     throw new Error("Comments plugin not found");
   }
 
-  const comments = editor.comments;
-
   const state = useUIPluginState(
     editor.comments.onUpdate.bind(editor.comments)
   );
-
   const selectedThreadId = state?.selectedThreadId;
 
   const threads = useThreads(editor);
 
-  const filteredThreads = Array.from(threads.values()).filter((thread) =>
-    props.filter === "resolved" ? thread.resolved : !thread.resolved
-  );
+  useEffect(() => {
```

Comment:
I'm not sure if this should be the responsibility of the sidebar per-se. Let's leave it for now and I'll review this in depth when the rest is ready to go